### DISCUSSION TITLE
Hurl Is it supported to execute a specified use case after it fails

### DISCUSSION THREAD

**Discussion by baerwang** [Orange-OpenSource/hurl#2816] [Q&A]
Some APIs create data, fail to get data, and how to delete the data after the fetch fails, such as specifying [after]

**Comment by baerwang** [comment]
POST {{host}}/objects
Authorization: Bearer {{access_token}}
Content-Type: application/json
```json
{"name":"example"}
```
HTTP 201
[Captures]
uuid: jsonpath "$.result.uuid"
[Asserts]
jsonpath "$.success" == true

GET {{host}}/objects/{{uuid}}
Authorization: Bearer {{access_token}}
HTTP 200
[Asserts]
jsonpath "$.success" == true
jsonpath "$.result.name" == "example1"  # `asserts failed`
[After] # `important point`
DELETE {{host}}/objects/{{uuid}}
Authorization: Bearer {{access_token}}

**Comment by fabricereix** [comment]
We can add the cli option `--continue-on-error` in the `[Options]` section


    POST {{host}}/objects
    Authorization: Bearer {{access_token}}
    Content-Type: application/json
    ```json
    {"name":"example"}
    ```
    HTTP 201
    [Captures] 
    uuid: jsonpath "$.result.uuid"
    [Asserts]
    jsonpath "$.success" == true

    GET {{host}}/objects/{{uuid}}
    Authorization: Bearer {{access_token}}
    [Options]
    continue-on-error: true
    HTTP 200
    [Asserts]
    jsonpath "$.success" == true
    jsonpath "$.result.name" == "example1"  # `asserts failed`

    DELETE {{host}}/objects/{{uuid}}
    Authorization: Bearer {{access_token}}
```
